content/php-client/ce-resources/media-files.md

```diff
@@ -52,7 +52,7 @@ You can get more information about this method [here](/php-client/list-resources
 #### With a cursor
 
 This method allows to iterate the media files. It will automatically get the next pages for you.
-With this method, it's not possible to get the previous page, or getting the total number of products.
+With this method, it's not possible to get the previous page, or getting the total number of medias.
```

Comment:
media is already plural, you could say media files instead

Reply:
I prefere `Media files`

Reply:
you can gtm and trust me to fix it @NolwennP 😺

Reply:
Never! 😈